generate_parameter_library_py/generate_parameter_library_py/test/valid_parameters_with_none_type.yaml

```diff
@@ -0,0 +1,11 @@
+some_controller:
+  some_parameter: {
+    type: string,
+    default_value: "default",
+    description: "Test parameter that should generate code.",
+  }
+
+  some_external_parameter: {
+    type: none,
```

Comment:
I understand and agree that we use internally `none` value, but in YAML, I would tend to use something more descriptive, e.g., `ExternalParameterStructure`, `ParameterStructure`, or something similar.

@tylerjw what do you think?

Reply:
I'm ok with using the none value in the yaml. Do you mind adding documentation to the readme about this feature and its intended use?

Reply:
done